clang/lib/CodeGen/CGCleanup.cpp

```diff
@@ -295,18 +295,25 @@ void EHScopeStack::Cleanup::anchor() {}
 static void createStoreInstBefore(llvm::Value *value, Address addr,
                                   llvm::Instruction *beforeInst,
                                   CodeGenFunction &CGF) {
-  auto store = new llvm::StoreInst(value, addr.emitRawPointer(CGF), beforeInst);
+  auto store = new llvm::StoreInst(value, addr.emitRawPointer(CGF),
+                                   beforeInst->getIterator());
```

Comment:
Is it worth updating the `beforeInst` parameter to a iterator instead of `->getIterator()`-ing here? That would match the `createLoadInstBefore` overload you've added in this patch too.